elements/drag-drop-elements/pl-drag-drop-element.js

```diff
@@ -0,0 +1,96 @@
+/* eslint-env jquery, browser */
+
+function set_answer(event) {
+    // We only care about when this function is fired
+    // from an ANSWER DROPZONE, aka dropzones with yellow backgrounds 
+    var textfield_name = event.target.getAttribute('name');
+    var dom_objs = $('#' + textfield_name + '-dropzone').children();
+    var student_answers = [];
+    var indents = [];
+    var answer_json = {'answers': [], 'answer_indent': []};
+    for (var i = 0; i < dom_objs.length; i++) {
+        if (!$(dom_objs[i]).hasClass('info')){
+            var answer_text = dom_objs[i].getAttribute('string');
+            var answer_indent = parseInt($(dom_objs[i]).css('marginLeft').replace('px', ''));
+            answer_indent = Math.round((answer_indent - 5) / 50); // get how many times the answer is indented
```

Comment:
If we are computing indent, it should refer to the same "tabwidth" variable mentioned below, so that changing a value in one place doesn't break code in the other place.

Reply:
will be fixed in next commit